### PR TITLE
Update babel config instructions for Babel 6

### PR DESCRIPTION
**Pull request by jedwards1211**
## Proposed changes

Fix https://github.com/webdriverio/webdriverio/issues/1485
Fix https://github.com/webdriverio/webdriverio/issues/1486

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation change

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [NA] I have added tests that prove my fix is effective or that my feature works (your grunt config for the tests is already using Babel 6 with `babel-runtime` as recommended in my documentation changes)
- [x] I have added necessary documentation (if appropriate)

### Reviewers: @christian-bromann

### CODE REVIEW

File: docs/guide/getstarted/configuration.md [hunk 1]
```diff
@@ -123,21 +123,22 @@ Default: *null*
...
 to register Babel in the before hook of your config file
 
 ```js
     before(function() {
-        require('babel-register')({
-            blacklist: [
-                'regenerator'
-            ]
-        });
+        require('babel-register');
```
Comment (christian-bromann) [inline docs/guide/getstarted/configuration.md]: should babel-register be in the npm install section or is it getting installed with babel-core?

**Comment by christian-bromann** [comment]
One comment otherwise LGTM

File: docs/guide/getstarted/configuration.md (hunk 1 above)
Comment (jedwards1211) [inline docs/guide/getstarted/configuration.md]: [It is](https://github.com/babel/babel/blob/master/packages/babel-core/package.json#L33), but a lot of other stuff does too that looks unnecessary?  Let me double check

**Comment by jedwards1211** [comment]
Okay, now i just recommended installing `babel-register`

**Comment by christian-bromann** [comment]
Sweet, that looks good 👍

**Comment by christian-bromann** [comment]
deployed to prod

**Pull request merged**